documentation/modules/ROOT/pages/transformations/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Transformations
+
+Connectors can be configured with transformations to make lightweight message-at-a-time modifications. {prodname} provides several single message transformations (SMTs) that you can use to modify event records before Kafka Connect saves the records to Kafka topics.
```

Comment:
Can you make "single message transformations" a link to upstream Kafka docs (look for the link prefix we have for that).

Reply:
SMTs can also be applied on the sink side (i.e. when reading from Kafka topics).

Reply:
Debezium Server also allows for using SMTs. We should not make it sound as if they only can be used with Kafka Connect.

Reply:
> Can you make "single message transformations" a link to upstream Kafka docs (look for the link prefix we have for that).

https://kafka.apache.org/documentation/#connect_transforms this one?

Reply:
> Debezium Server also allows for using SMTs. We should not make it sound as if they only can be used with Kafka Connect.

Second line is something i copied from https://debezium.io/documentation/reference/1.6/transformations/applying-transformations-selectively.html#_smt_predicates

"_Debezium provides several single message transformations (SMTs) that you can use to modify inbound event records before they are written to Kafka and outbound messages before they are sent to a sink connector._" 
Is this better or do you want me to remove the second line?

Reply:
> this one?

Yes, exactly. Just use the right variable instead of hard-coding the https://kafka.apache.org part.

Reply:
Let's go with "Debezium provides several single message transformations (SMTs) that you can use to either modify records before they are sent to Kafka (by applying them to the Debezium connectors) or when they are read from Kafka by a sink connector. Also Debezium Server (add link) supports the usage of SMTs).